test/python/unittest/API/APITest.py

```diff
@@ -146,25 +146,36 @@ def testData(self):
         # Get a particle species.
         electrons = i.particles["electrons"]
         self.assertIsInstance(electrons, openPMD.ParticleSpecies)
+        pos_x = electrons["momentum"]["x"]
+        w = electrons["weighting"][openPMD.Mesh_Record_Component.SCALAR]
```

Comment:
I wonder if we could realize this as well was `w = electrons["weighting"]` somehow